Dockerfile

```diff
@@ -132,8 +133,8 @@ EXPOSE 80 443
 
 LABEL name="manageiq" \
           vendor="ManageIQ" \
-          version="Capablanca" \
-          release="latest" \
+          version="Darga" \
+          release=${REF} \
```

Comment:
Guys, more details about image metadata/labels in Atomic Container Best Practices along with minimum recommended labels for an image : 

https://github.com/projectatomic/container-best-practices/blob/master/creating/creating_index.adoc